.github/workflows/create-token-pr.yaml

```diff
@@ -60,6 +52,7 @@ jobs:
         if: steps.pr-branch.outputs.exists == 'true'
         run: |
           git checkout ${{ steps.pr-branch.outputs.name }}
+          git merge origin/main -X ours --no-edit
           git merge ${{ github.ref_name }} -X theirs --no-edit
```

Comment:
@oliverschuerch I updated the workflow to work as follow: 
1. Check if the merging branch exist:
    - If **it does not exists** ⇨ create the merging branch from the tokens branch
    - if **it exists** ⇨ update the merging branch with both the main and the token branches (in case of conflicts accept changes from the tokens branch because it "is always king", thus the `-X theirs`)
2. Check if the PR exists:
    - If **it does not exists** ⇨ create it
    - If **it exists** ⇨ we are good to go